### PR TITLE
Create a layout for pages that occur when you add new features in a review

### PR DESCRIPTION
**Pull request by sinji2102**
## Describe changes

_Describe a summary of the changes and the related issue to communicate to the maintainers why we should accept this pull request._

## Issue number or link
#131 

## Types of changes

What is the type of code change?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (changes that resolve errors)
- [x] New feature (changes which adds functionality)
- [ ] Breaking change (big changes that affect existing functionality)
- [ ] Documentation Update

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the "README.md"
- [x] My code follows the style guidelines of this project
- [x] My changes generate no new warnings
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

_Please let me know if there's anything else to explain_

### CODE REVIEW

File: src/components/common/ReviewStepInfo.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+const ButtonWrapper = styled(Button)`
+  padding: 0.8rem 1.6rem 0.8rem 1.6rem;
+  height: 100%;
+  margin: 1rem;
+  font-size: 2rem;
+`;
+
+const Box_ = styled.div`
+  gap: 2rem;
+  display: flex;
```
Comment (ancy0) [inline src/components/common/ReviewStepInfo.js:100]: 😢

File: src/layout/ReviewLayout.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+import { Outlet } from "react-router-dom";
+import { useRecoilState } from "recoil";
+import Button from "@mui/material/Button";
+import { Header } from "./Header";
+import StepInfo from "../components/common/StepInfo";
+import { FinishDialog } from "../components/common/modal/FinishDialog";
+import { CancelDialog } from "../components/common/modal/CancelDialog";
+import { BaseDialog } from "../components/common/modal/BaseDialog";
+import { modalState } from "../recoil/commonState";
+import { useState } from "react";
```
Comment (ancy0) [inline src/layout/ReviewLayout.js:12]: 😢

File: src/assets/images/test.gif
Comment (ancy0) [inline src/assets/images/test.gif:1]: Where is this gif used?
If you don't need it anymore after using it for testing, you'd better delete the file.

**Review by ancy0 — COMMENTED** [review]
👍

**Review by hummingbbird — APPROVED** [review]
💯

**Pull request merged**